### PR TITLE
Work with net-scp 4.0 when it ships

### PR DESCRIPTION
**Pull request by tas50**
This will allow pulling in the new net-ssh which supports new algos

Signed-off-by: Tim Smith <tsmith84@gmail.com>

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,12 +10,6 @@ group :integration do
   gem "kitchen-vagrant"
 end
 
-group :debug do
-  gem "pry", "~>0.12"
```
Comment (tas50) [inline Gemfile:14]: New ruby replaces pry so nuke all this

**Review by vkarve-chef — APPROVED** [review]

**Comment by Stromweld** [comment]
net-scp just officially release 4.0 to rubygems. Is there anything that needs to be done to build a new release of test-kitchen with the latest net-scp and net-ssh to fix these ssh login issues with the latest releases of rhel and ubuntu?

**Comment by tas50** [comment]
It's being held back by train from the InSpec team. They need to update the dep on net-scp and release train.

**Comment by clintoncwolfe** [comment]
done - https://rubygems.org/gems/train-core/versions/3.10.7

**Pull request merged**